devicetypes/smartthings/smartsense-button.src/smartsense-button.groovy

```diff
@@ -27,7 +27,7 @@ metadata {
         capability "Health Check"
         capability "Sensor"
 
-        fingerprint inClusters: "0000,0001,0003,0020,0402,0500", outClusters: "0019", manufacturer: "Samjin", model: "button", deviceJoinName: "Button"
+        fingerprint inClusters: "0000,0001,0003,0020,0402,0500", outClusters: "0019", manufacturer: "Samjin", model: "button", deviceJoinName: "SmartThings Button" //Button
```

Comment:
Most of the SmartThings devices have join name translations in the `messages.properties` file associated with the DTH. Thus any change to the join name must also come with updates to that file. The translations themselves likely do not/should not change, but the starting string that is translated does need to be updated.